integration_tests/test_employment.py

```diff
@@ -45,3 +45,20 @@ def test_employed_have_income(tracked_live_populations, time_step):
     # All people in military group quarters are
     employed = pop[pop["employer_id"] != data_values.UNEMPLOYED_ID]
     assert (employed["income"] > 0).all()
+
+
+def test_movers_change_employment(tracked_live_populations):
+    for before, after in zip(tracked_live_populations, tracked_live_populations[1:]):
```

Comment:
Add a comment explaining which populations you're comparing

Reply:
Do you mean a comment explaining why I am using the tracked live population instead of total population? I actually think this would be a slightly better check if I use the total population, so I'll switch to that.

Reply:
It's a bit of a tension between making tests sensitive and making them non-overlapping. I used the living population here because we already have a test that checks that only living people move. But, I think it's fine to have overlapping tests especially if it makes it harder to miss an issue.